files/default/neutron-ha-tool.py

```diff
@@ -623,7 +623,8 @@ def migrate_router_safely(qclient, noop, router_id, agent, target,
         return True
 
     try:
-        migrate_router(qclient, router_id, agent, target,
+        router = get_router(qclient, router_id)
```

Comment:
I would really prefer if we could avoid  this additional API queries. ```list_routers_on_l3_agent``` already gets all the required attributes as a result. So if we could change it to return a list of routers instead of just router_ids it should be possible. (IIRC all the callers of ```migrate_router_safely``` process results ```list_routers_on_l3_agent```)